routes/web.php

```diff
@@ -135,7 +135,10 @@
         'as'=>'vendors.'
     ], function(){ 
         Route::get('/vendorPage',[VendorsController::class,'index']);
-        Route::get('/individualVendor',[VendorsController::class,'show']);
+        Route::get('/individualVendor/{id}',[VendorsController::class,'edit'])->name('vendors.edit');
+        Route::post('/individualVendor/{id}',[VendorsController::class,'update'])->name('vendors.update');
+        Route::delete('/individualVendor/{id}',[VendorsController::class,'destroy'])->name('vendors.destroy');
+        Route::post('/vendorPage', [VendorsController::class, 'store']);
```

Comment:
Does `/vendors/` URI work on this? I recall `vendor` doesn't work due to conflicts with `vendor` folder. Was thinking of it as a replacement instead of `/individualVendor/*` and `/vendorPage`

Reply:
Yes sir, I think /vendors/ is better. I'll make the changes once it's merged.

Reply:
Oh right, I think its more proper to create a separate PR for this. Just referenced this to a new issue.
https://github.com/justinmanigo/HMEZGEB/issues/29